server/webserver.go

```diff
@@ -692,22 +692,6 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 
 	mux.HandleFunc("/claimEarnings", func(w http.ResponseWriter, r *http.Request) {
 		if s.LivepeerNode.Eth != nil {
-			if err := r.ParseForm(); err != nil {
-				glog.Errorf("Parse Form Error: %v", err)
-				return
-			}
-
-			endRoundStr := r.FormValue("endRound")
-			if endRoundStr == "" {
-				glog.Errorf("Need to provide amount")
-				return
-			}
-			endRound, err := lpcommon.ParseBigInt(endRoundStr)
```

Comment:
Seems like it might be ok to allow a user to still specify an endRound. Don't imagine users needing to do this, but if they have tools that depend on being able to do this for whatever reason it might be good to not break that expectation.

Reply:
I don't think there's much tooling using this, however it might be okay to force a user's hand in changing their expectation of how this functionality truly behaves currently. If those expectations are not altered somehow it will just end up incurring additional costs for those users and their expectations misalign with what optimal behaviour would be.

Reply:
I believe there are a few users that have automate claimEarnings calls via this webserver endpoint in the past. Generally, we should avoid breaking API compatibility and we should avoid making assumptions around how users use an API when we can, but I see your point about a user not realizing that there is basically no benefit in calling claimEarnings with an endRound set to anything besides the current round. Will leave this case up to you.

Reply:
I actually think it's fine because the API itself doesn't change. Users can still call the same endpoint with an argument (it will just be ignored).

If there are tools out there that automate claimEarnings they have become rather obsolete anyway even it's for up until the current round. 

If a user needs funds it's more economical to perform a  staking action for example unbond, or withdraw fees directly.
From that perspective it make even make sense to just remove the endpoint entirely, while it might misalign with user expectations (we can clarify this in the release notes) it protects users from operating inefficiently.